listenbrainz/webserver/static/js/tests/utils/utils.test.tsx

```diff
@@ -54,6 +57,7 @@ describe("preciseTimestamp", () => {
       new Date(testDate).toLocaleString(undefined, {
         day: "2-digit",
         month: "short",
+        year: "numeric",
```

Comment:
Why was there a need to change the test assertion?
We need to be careful of modifying tests in such a huge PR with so many changes.
Is it definitely the same result? If so a PR comment would be welcome.